style.css

```diff
@@ -240,3 +288,76 @@ textarea {
 #form button {
   margin-right: 211px;
 }
```

Comment:
- [ ] According to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), the "Get In Touch" button is supposed to be placed at the centre of the form 
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/166070543-1f572a2a-e46f-4e5e-93bc-eb23e3faa353.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/166070640-19ba48e5-4fb6-4ec2-8420-a3c609945b53.png)